docs/_includes/common/wizard-stepper.html

```diff
@@ -1,72 +1,58 @@
-<div id="select">
+<div id="wizard-stepper">
     {% include common/section-header.html name="wizard-stepper" version=page.versions.wizard-stepper %}
 
     <p>TODO</p>
 
-    <div id="wizard-stepper">
-        <div class="wizard-stepper-list">
-            <div class="wizard-step-container--confirmation">
-                <div class="wizard-step">
-                    <svg class="icon icon--confirmation-filled" focusable="false" height="24" width="24">
-                        <use xlink:href="#icon-confirmation-filled"></use>
-                    </svg>
-                    <span class="wizard-step-title">
-                        Started
-                    </span>
-                </div>
+    <ul class="wizard-stepper-list">
```

Comment:
There can be only one base class in a module. I currently see two: `wizard-stepper` and `wizard-stepper-list`. So we have to decide on one.

The reason I wanted to go with `wizard-stepper-list` (which is quite long, I know) was to differentiate it from the other type of wizard stepper that uses cards. However... I now think that we can just use `wizard-stepper` as the base class for this version you are working on here. It feels like this version will be the most often used one, and so we can consider it the default.

This would mean changing:

* `wizard-stepper-list` to `wizard-stepper__items`
* `wizard-stepper-list__item--confirmation` to `wizard-stepper__item wizard-stepper__item--confirmation`
* etc

Basically removing the word list from everywhere. Happy to wait and go through this in person if it would make more sense.

I also had an idea where we might be able to use this same structure for the other type of wizard stepper and drop card-type steps into the list, rather than icon-and-line-type steps. But let's get this version done first and see if that would work later... :-)

Reply:
@ianmcburnie I do not have a class called `wizard-stepper` which you are referring as another base class. My base class is `wizard-stepper-list` which I thought would go hand in hand with HTML list (ul, ol) and list items (li)

Reply:
Oh my bad, it's an ID not a class. Let me re-review this aspect...

Reply:
I guess I got confused because the example is missing the usual containing structure of `<div class="demo"><div class="demo__inner">..</div></div>`. Could you please add that? Adding the example markup right afterwards would be useful too.

Reply:
So I'm thinking something like this would get us closer to our ideal HTML structure and BEM naming:

```
<div class="wizard-stepper">
    <ul class="wizard-stepper__items">
        <li class="wizard-stepper__item wizard-stepper__item--confirmation">
            <span class="wizard-stepper__icon">
                <svg class="icon icon--confirmation-filled" focusable="false" height="24" width="24">
                    <use xlink:href="#icon-confirmation-filled"></use>
                </svg>
            </span>
            <span class="wizard-stepper__text">Started</span>
        </li>
        ..
    </ul>
</div>
```

Vertical might only need a base class modifier:

```
<div class="wizard-stepper wizard-stepper--vertical">
    <!-- same as above example -->
</div>
```

We could possibly use a similar structure for the card/form based stepper (when we get to it later):

```
<div class="wizard-stepper">
    <ul class="wizard-stepper__cards">
        <li class="wizard-stepper__card wizard-stepper__card--confirmation">
            ..
        </li>
        ..
    </ul>
</div>
```

Although we may end up only just needing something as simple as this, with no list structure, we shall see:

```
<div class="wizard-card">
    ..
</div>
```

Let's discuss further in our meeting today.